bridge_reachability/tor-bridges-ip-port.csv

```diff
@@ -152,153 +152,160 @@ host,port,nickname,protocol
 5.196.23.64,9001,Aerodynamik01,or_port
-81.30.158.213,9001,dumpster,or_port
-104.200.20.46,9001,bynumlawtor,or_port
-62.210.129.246,443,MilesPrower,or_port
-82.223.21.74,9001,silentrocket,or_port
 51.254.136.195,443,torproxy02,or_port
-77.247.181.162,443,sofia,or_port
 185.100.84.82,443,saveyourprivacyexit,or_port
-199.249.223.69,443,Quintex20,or_port
 193.11.114.45,9002,mdfnet2,or_port
+51.254.96.208,9001,rofltor01,or_port
+217.12.199.190,443,torpidsUAitlas,or_port
+192.42.116.16,443,hviv104,or_port
+192.160.102.164,9001,snowfall,or_port
+192.87.28.82,9001,Smeerboel,or_port
 62.210.254.132,443,turingmachine,or_port
-85.230.184.93,443,Logforme,or_port
-72.52.75.27,9001,piecoopdotnet,or_port
 185.96.88.29,443,TykRelay05,or_port
-176.10.104.243,443,DigiGesTor2e1,or_port
+93.180.156.84,9001,BARACUDA,or_port
+51.15.205.214,9001,titania1,or_port
 163.172.194.53,9001,GrmmlLitavis,or_port
-5.189.169.190,8080,thanatosDE,or_port
-151.80.42.103,9001,matlink,or_port
+81.7.11.96,9001,Doedel22,or_port
 37.187.20.59,443,torpidsFRovh,or_port
-62.138.7.171,8001,0x3d005,or_port
-178.16.208.57,443,bakunin,or_port
-91.219.237.244,443,lewwerDuarUesSlaav,or_port
+51.255.41.65,9001,PrisnCellRelayFR1,or_port
+54.37.73.111,9001,Aerodynamik05,or_port
+96.253.78.108,443,NSDFreedom,or_port
+193.234.15.57,443,bakunin,or_port
+204.8.156.142,443,BostonUCompSci,or_port
 37.153.1.10,9001,smallsweatnode,or_port
-163.172.223.200,443,Outfall2,or_port
+173.212.254.192,31337,ViDiSrv,or_port
 91.229.20.27,9001,gordonkeybag,or_port
 66.111.2.20,9001,NYCBUG0,or_port
 185.100.86.128,9001,TorExitFinland,or_port
-146.185.177.103,9030,Winter,or_port
-199.249.223.64,443,Quintex15,or_port
+86.105.212.130,443,firstor2,or_port
+31.185.104.19,443,Digitalcourage3ip1,or_port
 46.28.110.244,443,Nivrim,or_port
-91.121.84.137,4052,lindon,or_port
 46.165.230.5,443,Dhalgren,or_port
 171.25.193.77,443,DFRI1,or_port
+87.118.122.120,443,otheontelth,or_port
 81.7.3.67,443,BeastieJoy62,or_port
 171.25.193.78,443,DFRI4,or_port
-178.16.208.58,443,jaures2,or_port
-163.172.149.122,443,niij03,or_port
+193.234.15.58,443,jaures2,or_port
+128.31.0.13,443,csailmitexit,or_port
+94.142.242.84,443,rejozenger,or_port
 195.154.164.243,443,torpidsFRonline3,or_port
 86.59.119.88,443,ph3x,or_port
 185.129.62.62,9001,kramse,or_port
 188.40.128.246,9001,sputnik,or_port
-176.126.252.11,9001,chulak,or_port
+31.185.104.20,443,Digitalcourage3ip2,or_port
+45.79.108.130,9001,linss,or_port
 5.9.147.226,9001,zwiubel,or_port
 178.17.174.14,9001,TorExitMoldova,or_port
-199.249.223.40,443,Quintex31,or_port
 212.129.62.232,443,wardsback,or_port
 136.243.214.137,443,TorKIT,or_port
-212.47.233.86,9001,netimanmu,or_port
+193.234.15.60,443,jaures3,or_port
 93.115.97.242,9001,firstor,or_port
 81.2.209.10,80,torzabehlice,or_port
+185.220.101.32,20032,niftypika,or_port
 193.11.114.46,9003,mdfnet3,or_port
+85.248.227.164,9002,tollana,or_port
 81.7.11.186,443,BeastieJoy60,or_port
-197.231.221.211,443,IPredator,or_port
```

Comment:
I am not 100% sure we want to be deleting IPs of bridges that we used to test (even if they are now offline).

This bridge in particular still seems to work, so it's probably wise to keep it in there.

Reply:
This is an (old) fallback directory mirror. We update the fallback list every 6-12 months as relays fail (or change address or keys). We also rotate fallbacks to evade censorship.

We'd like ooni to test the latest fallbacks, authorities, and Tor Browser bridges. Testing old fallbacks, authorities, and bridges isn't that helpful, because most Tor clients and Tor Browser installs are up to date.

And you'll get spurious failures, because we remove some fallbacks and bridges *because* they have failed.